metricbeat/module/munin/node/_meta/docs.asciidoc

```diff
@@ -0,0 +1,34 @@
+=== munin node MetricSet
+
+This is the node metricset of the module munin.
+
+[float]
+=== Features and configuration
+
+The node metricset of the munin module collects metrics from a munin node agent
+and sends them as events to Elastic.
+
+[source,yaml]
+---
+- module: munin
+  metricsets: ["node"]
+  hosts: ["localhost:4949"]
+---
+
+All metrics exposed by a single munin node will be sent as an only event,
```

Comment:
"All metrics exposed by a single munin node will be sent in a single event, grouped by munin items." ?

An example here would be nice.